_docs/quickstarts/scanner/descriptions.md

```diff
@@ -74,6 +90,20 @@ Inventory pipelines:
 
 - `load_forwarding_rules_pipeline`
 
+## GKE Version scanner
```

Comment:
Google Kubernetes Engine (GKE)...

Reply:
Branding for this is just "Kubernetes Engine": it doesn't have a "Google" prefix and we shouldn't use GKE. Please update all occurrences of GKE in title and text, thanks!

Reply:
Updated.  Will update the link when joe is done with the global renaming.